lib/srv/authhandlers.go

```diff
@@ -335,18 +335,15 @@ func (h *AuthHandlers) fetchRoleSet(cert *ssh.Certificate, ca services.CertAutho
 	// for local users, go and check their individual permissions
 	var roles services.RoleSet
 	if clusterName == ca.GetClusterName() {
-		users, err := h.AccessPoint.GetUsers()
+		// Fetch user from cache.
```

Comment:
the fact that it's a cache is implementation and configuration dependent, as far as the code is concerned, this is just a client point.

Reply:
(so remove the comment)